packages/paste-website/src/components/homepage/ComponentsAndPatterns.tsx

```diff
@@ -0,0 +1,172 @@
+    case 'Create':
+      return <HomeCreateIllustration />;
+    case 'Delete':
+      return <HomeDeleteIllustration />;
+    default:
+      return <HomeButtonIllustration />;
+  }
+};
+
+interface ComponentCardProps {
+  component: string;
+  url: string;
+}
+
+const ComponentCard: React.FC<ComponentCardProps> = ({component, url}) => {
+  return (
+    <Box
+      backgroundColor="colorBackgroundBody"
+      borderColor="colorBorderLighter"
+      borderRadius="borderRadius20"
+      borderStyle="solid"
+      borderWidth="borderWidth20"
+      marginBottom="space60"
+      padding="space50"
+      textAlign="center"
+    >
+      <Heading as="h4" variant="heading40">
+        {component}
+      </Heading>
+      {renderComponentCardIllustration(component)}
+      <Text as="span" fontWeight="fontWeightSemibold">
+        <Anchor href={url}>See {component}</Anchor>
+      </Text>
+    </Box>
+  );
+};
+
+const ComponentsAndPatterns: React.FC = () => {
+  const theme = useTheme();
+  return (
+    <Box
+      backgroundColor="colorBackgroundBody"
+      borderRadius="borderRadius20"
+      margin="space70"
+      position="relative"
+      overflow="hidden"
+    >
+      <Box backgroundColor="colorBackground" padding={['space90', 'space180']}>
+        <Box
+          color="colorTextWeak"
+          fontWeight="fontWeightBold"
+          marginBottom="space110"
+          position="relative"
+          _after={{
+            borderTopColor: 'colorBorderLight',
+            borderTopStyle: 'solid',
+            borderTopWidth: 'borderWidth10',
+            content: `""`,
+            display: 'block',
+            height: '1px',
+            marginLeft: 'space160',
+            position: 'absolute',
+            top: '50%',
+            width: 'calc(100% - 3.75rem)', // 100% - space160
+          }}
+        >
+          Popular
+        </Box>
+        <Grid gutter="space60">
+          <Column span={[12, 6]}>
+            <Box textAlign="center">
+              <Box
+                alignItems="center"
+                backgroundColor="colorBackgroundBody"
+                borderRadius="borderRadiusCircle"
+                display="flex"
+                height="sizeSquare150"
+                justifyContent="center"
+                marginBottom="space40"
+                marginLeft="auto"
+                marginRight="auto"
+                width="sizeSquare150"
+              >
+                <ComponentsIcon
+                  css={{height: theme.heights.sizeIcon40, width: theme.widths.sizeIcon40}}
+                  color={theme.textColors.colorTextBrandHighlight}
+                  decorative
+                />
+              </Box>
+              <Heading as="h2" variant="heading20">
+                Components
+              </Heading>
+            </Box>
+            <Grid gutter="space30">
+              <Column span={[12, 6]}>
+                <ComponentCard component="Button" url="/components/button" />
```

Comment:
I'd personally go with a more compositional approach than this, passing it the children it needs, like the illustration